src/Database/LSMTree/Internal/Index/OrdinaryAcc.hs

```diff
@@ -7,10 +9,7 @@
 module Database.LSMTree.Internal.Index.OrdinaryAcc
 (
     IndexOrdinaryAcc,
-    new,
-    appendSingle,
-    appendMulti,
-    unsafeEnd
```

Comment:
Same comment about keeping the exports

Reply:
Please see [my below general comment on specialization](https://github.com/IntersectMBO/lsm-tree/pull/476#issuecomment-2491953851).